applications/sasquatch-backpack/values.yaml

```diff
@@ -110,3 +110,10 @@ redis:
 
   # -- Affinity rules for the Redis pod
   affinity: {}
+
+cluster:
+  name: backpack
+
+strimzi:
+  enabled: true
```

Comment:
I would just list the Kafka topics you want to create here, assuming that Strimzi is deployed.